CONTRIBUTORS

```diff
@@ -0,0 +1,2 @@
+Halil Erdoğan	herdogan AT sabanciuniv DOT edu 
```

Comment:
@hedonistrh should we use another e-mail? :)

Reply:
We can use herdoganturkey@gmail.com 💯 :)